hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/MixedValidationsAfterReconnect.java

```diff
@@ -112,9 +112,9 @@ private HapiSpec validateTopicInfo() {
     }
 
     private HapiSpec validateFileInfo() {
-        String firstlyCreatedFile = "0.0.41363";
-        String lastlyCreatedFile = "0.0.42362";
-        String invalidFileId = "0.0.42363";
+        String firstlyCreatedFile = "0.0.41064";
+        String lastlyCreatedFile = "0.0.42063";
+        String invalidFileId = "0.0.42064";
```

Comment:
Can these numbers be defined as constant, or if have dependency on property files, should be read from property files.

Reply:
Extracted to constants since we don't read from any properties files in this test